src/components/HeroSection.tsx

```
@@ -0,0 +1,38 @@
+                borderRadius='1.75rem'
+                className="bg-white dark:bg-slate-900 text-black dark:text-white border-neutral-200 dark:border-slate-800">
+                Getting Started
+                </Button>
+                </Link>
+        </div>
+               
+          
+      </div>
+    </div>
+  );
+}export default SpotlightNewDemo;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)